src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+h1 {
+  font-weight: bold;
+  font-size: 17px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+p {
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
fixed